redwood-lazylayout-compose/src/commonTest/kotlin/app/cash/redwood/lazylayout/compose/LazyListTest.kt

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.lazylayout.compose
+
+import androidx.compose.runtime.Composable
+import app.cash.redwood.protocol.ChildrenChange
+import app.cash.redwood.protocol.ChildrenTag
+import app.cash.redwood.protocol.Create
+import app.cash.redwood.protocol.Id
+import app.cash.redwood.protocol.ModifierChange
+import app.cash.redwood.protocol.PropertyChange
+import app.cash.redwood.protocol.PropertyTag
+import app.cash.redwood.protocol.WidgetTag
+import app.cash.redwood.testing.toChangeList
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import example.redwood.compose.ExampleSchemaProtocolBridge
+import example.redwood.compose.Text
+import example.redwood.widget.ExampleSchemaTester
+import kotlin.test.Test
+import kotlinx.coroutines.test.runTest
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+
+class LazyListTest {
+  @Test
+  fun emptyLazyColumn() = runTest {
+    val content = @Composable {
+      LazyColumn(placeholder = { Text("Placeholder") }) {
+      }
+    }
+
+    val snapshot = ExampleSchemaTester {
+      setContent(content)
+      awaitSnapshot()
+    }
+
+    val expected = listOf(
```

Comment:
This is a bit unwieldy

Reply:
Yeah I'm curious if this is the right layer at which to test this behavior, as opposed to the normal tester and asserting against values.

Reply:
I think the normal tester was what I wanted. The protocol level was definitely too low for this sorta thing.